flask/README.rst

```diff
@@ -59,7 +61,25 @@ Start flask web server:
     $ cd YOUR_PROJECT_DIR/flask/
     $ python runserver.py
 
-Go to http://127.0.0.1:5000/ in your browser
+Go to http://127.0.0.1:5000/ in your browser.
+
+If you do like that, all data added will be stored in a in-memory SQLite database.
+If you want to keep the state, export the environment variable ENVIRONMENT:
+
+.. code-block:: bash
+
+    $ export ENVIRONMENT=development
+    $ python runserver.py
+
+Or in a single line:
+
+.. code-block:: bash
+
+    $ ENVIRONMENT=development python runserver.py
```

Comment:
Let's keep it simple and leave only this single line version.

Reply:
Done!